src/core/MOM.F90

```diff
@@ -555,6 +555,7 @@ module MOM
   integer :: id_zossq    = -1
   integer :: id_volo     = -1
   integer :: id_ssh      = -1
+  integer :: id_ssh_ga   = -1
```

Comment:
They are already implemented as sosga and tosga.

--------------------------------
Dr. John Krasting
Physical Scientist (NOAA Federal)
NOAA/Geophysical Fluid Dynamics Laboratory
Biogeochemistry, Ecosystems, and Climate Group
Princeton University Forrestal Campus
201 Forrestal Road
Princeton, NJ 08540

P. (609) 452-5359
F. (609) 987-5063

On Thu, Aug 13, 2015 at 3:33 PM, Alistair Adcroft <notifications@github.com>
wrote:

> In src/core/MOM.F90
> <https://github.com/NOAA-GFDL/MOM6/pull/215#discussion_r37015843>:
>
> > @@ -555,6 +555,7 @@ module MOM
> >    integer :: id_zossq    = -1
> >    integer :: id_volo     = -1
> >    integer :: id_ssh      = -1
> > +  integer :: id_ssh_ga   = -1
>
> Could you add id_sst_ga and id_sss_ga while you're at it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NOAA-GFDL/MOM6/pull/215/files#r37015843>.
>